x-pack/plugins/canvas/canvas_plugin_src/uis/datasources/essql.js

```diff
@@ -77,7 +77,7 @@ class EssqlDatasource extends PureComponent {
           </EuiText>
         }
       >
-        <CodeEditorField
+        <CodeEditorField // TODO theme appears to be provided in x-pack/plugins/canvas/public/application.tsx
```

Comment:
Dark theme looks good here:

The white on the left side is the actual contents of the Canvas workpad which is not intended to update.

<img width="418" alt="Screen Shot 2023-06-22 at 6 12 30 PM" src="https://github.com/elastic/kibana/assets/14276393/287eb9de-1205-4a87-9675-abe148996ae9">

Reply:
Ah ok - thanks!